openshift/app.dc.yaml

```diff
@@ -162,6 +162,36 @@ objects:
                     secretKeyRef:
                       key: password
                       name: getok-sc-ches-secret
+                - name: SC_KC_DEV_USERNAME
+                  valueFrom:
+                    secretKeyRef:
+                      key: username
+                      name: getok-sc-keycloakdev-secret
```

Comment:
I think we should have this be `getok-sc-keycloak-dev-secret` instead (and updated for everything else referencing it in order to be consistent with all the other secret nomenclatures.

Reply:
Ok. I can wrap this as a part of the upcoming db seed changes as that will do some pipeline modifications anyways.